src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py

```diff
@@ -60,14 +65,19 @@ def test_containerapp_identity_e2e(self, resource_group):
     @AllowLargeResponse(8192)
     @ResourceGroupPreparer(location="canadacentral")
     def test_containerapp_identity_system(self, resource_group):
+        location = os.getenv("CLITestLocation")
+        if not location:
+            location = 'eastus'
+        self.cmd('configure --defaults location={}'.format(location))
```

Comment:
May I ask whether we can abstract the logic of setting default test location into a reusable public `@LocationPreparer`?

Reply:
If you can have a LocationPreparer, we could refine our code with LocationPreparer in the future.